tools/gfx/d3d11/render-d3d11.cpp

```diff
@@ -51,7 +50,7 @@ using namespace Slang;
 
 namespace gfx {
 
-class D3D11Renderer : public GraphicsAPIRenderer
+class D3D11Renderer : public ImmediateRendererBase
```

Comment:
Oh, never mind, you did introduce a base class, and it is just that CUDA was different enough to not warrant using it. Good.